client/src/locales/en.json

```diff
@@ -68,6 +68,10 @@
   "authorization.auth-box.sign-in": "Sign in with:",
   "authorization.auth-box.cta": "Try it out:",
   "authorization.auth-box.demo-button": "Live demo",
+  "authorization.reset-password.heading": "Reset your password",
+  "authorization.reset-password.email-label": "Enter email",
+  "authorization.reset-password.tip-content": "Please enter correct email",
```

Comment:
Please read this article:
https://uxplanet.org/error-message-guidelines-6ce257d3d0bd

Maybe text: "Please enter a valid emila address" would be nicer :)